src/main/java/it/pagopa/pm/gateway/dto/XPayPollingResponse.java

```diff
@@ -9,9 +9,12 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public class XPayAuthPollingResponse {
+public class XPayPollingResponse {
 
     private String status;
```

Comment:
what are the possible values?

Reply:
Possible combinations of these parameters in the response are documented in design review: https://pagopa.atlassian.net/wiki/spaces/TKM/pages/541589608/DR+-+XPAY+-+Richiesta+autorizzazione+PM+-+PGS#%5BGET%5D-request-payments%2Fxpay%2F%7BrequestId%7D